ui/public/locales/en.json

```diff
@@ -668,15 +668,12 @@
 "label.duration.7days": "7 days",
 "label.dynamicscalingenabled": "Dynamic scaling enabled",
 "label.dynamicscalingenabled.tooltip": "VM can dynamically scale only when dynamic scaling is enabled on template, service offering and global setting.",
-<<<<<<< HEAD
 "label.diskofferingstrictness": "Disk offering strictness",
```

Comment:
Thanks for the PR @nvazquez . There are duplicates also, can you please check them too.

Reply:
@harikrishna-patnala this is the only place I can see the `HEAD`; do you see other places in the UI or backend code with merge conflicts?